lms/fixtures/custom_field.json

```diff
@@ -2012,5 +2012,58 @@
   "translatable": 0,
   "unique": 0,
   "width": null
- }
+ },
+ {
+    "allow_in_quick_entry": 0,
+    "allow_on_submit": 0,
+    "bold": 0,
+    "collapsible": 0,
+    "collapsible_depends_on": null,
+    "columns": 0,
+    "default": null,
+    "depends_on": null,
+    "description": null,
+    "docstatus": 0,
+    "doctype": "Custom Field",
+    "dt": "LMS Course",
```

Comment:
@Rola-anvil why add a custom field in LMS Course doctype when you can add a standard field.

Reply:
Also, you have made the changes in the main branch of your fork. Can you create another branch for this feature instead?